uqcsbot/starboard.py

```diff
@@ -232,13 +275,37 @@ def _rm_big_ratelimit(self, id: int) -> None:
         """Callback to remove a message from the big-ratelimited list"""
         self.big_blocked_messages.remove(id)
 
-    def _starboard_db_add(self, recv: int, recv_location: int, sent: int) -> None:
+    def _starboard_db_add(self, recv: int, recv_location: int) -> None:
         """Creates a starboard DB entry. Only called from _process_updates when the messages are not None, so doesn't
```

Comment:
Potentially mention that `_starboard_db_finish` should be run at some point after this method.